<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignupPage</class>
 <widget class="QWidget" name="SignupPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tutorify - Signup</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f9fcff;</string>
  </property>
  <widget class="QLabel" name="labelSignup">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px; font-weight: bold; color: #1a3c57;</string>
   </property>
   <property name="text">
    <string>Create Your Tutorify Account</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14px; color: #1a3c57;</string>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineName">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter your full name</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEmail">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14px; color: #1a3c57;</string>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEmail">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter your email</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14px; color: #1a3c57;</string>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePassword">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Enter your password</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCNIC">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14px; color: #1a3c57;</string>
   </property>
   <property name="text">
    <string>CNIC:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCNIC">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter your CNIC</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRole">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14px; color: #1a3c57;</string>
   </property>
   <property name="text">
    <string>Role:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboRole">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Student</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tutor</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnSignup">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ef476f; color: white; font-size: 14px; border-radius: 8px; padding: 5px;</string>
   </property>
   <property name="text">
    <string>Signup</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLoginRedirect">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>260</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 12px; color: #3c7aa6;</string>
   </property>
   <property name="text">
    <string>Already have an account? Login here.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>